### DISCUSSION TITLE
사이트 우회 문의

### DISCUSSION THREAD

**Discussion by swbaek81** [List-KR/community#52] [Q&A]
안녕하세요 
윈도우10 애드가드프로(7.16.0[4542]) 엣지브라우저 인데요
http://www.cookmana18.com/ 
사이트에서 만화보려고 하면 아래와 같이 안되서요 (스텔스 모드 - DPI 로 부터 보호 체크 상태)
Adguard VPN 켜니까 OK되더라고요 

애드가드VPN 외에는 방법 없을까요?
![image](https://github.com/List-KR/community/assets/15683789/0302a65a-b1f8-4477-a4bc-b392f13d46c6)

**Comment by piquark6046** [comment]
HTTP 통신을 하는 것 같은데 웹 사이트가 HTTPS를 안 지원하나요?

**Reply by swbaek81** [reply]
![image](https://github.com/List-KR/community/assets/15683789/3c91b1e6-9d16-4cd5-9dd4-0aaad815ad15)
 
위와 같은데 지원 안하는 걸까요?

**Reply by piquark6046** [reply]
LG U+ 에선 AdGuard의 DPI로 부터 보호 비활성화되어 있어도 잘 연결됩니다.
어떤 통신사를 사용하시나요?

**Reply by swbaek81** [reply]
윈도우OS 데스크탑 환경에 유선 인터넷 이에요

**Reply by piquark6046** [reply]
그럼, 테스트를 위해서 AdGuard for Windows를 시스템 트레이를 통해 AdGuard 서비스까지 종료하신 후에 유니콘 HTTPS를 설치 및 활성화하셔서 다시 연결을 시도해보시겠어요?

**Reply by swbaek81** [reply]
얘기 주신대로 하니까 잘 되네요

**Reply by piquark6046** [reply]
AdGuard의 DPI로 부터 방지 기능은 안되고요?
KT 유선 네트워크를 사용하시나요?

**Reply by swbaek81** [reply]
AdGuard의 DPI로 부터 방지 기능은 안되고요? -> 네 맞습니다.

SK브로드밴드 케이블 유선 인터넷 사용 중입니다

**Reply by swbaek81** [reply]
이 상태 인데, 지금 이 사이트는 애드가드 로는 안되요
![image](https://github.com/List-KR/community/assets/15683789/cd9cfe95-4ece-43a6-a05d-b9e2aacacb61)

**Reply by piquark6046** [reply]
Android에 비해 Windows는 AdGuard VPN외 다른 VPN 사용에 제한적이지 않은 것으로 알고 있습니다.
https://github.com/AdguardTeam/CoreLibs/issues/1789 가 완료되기 전 까지 AdGuard for Windows와 Proton VPN 무료 서버(일본)를 함께 사용해보시는 것은 어떻겠습니까?

**Reply by piquark6046** [reply]
아니면, 위 issue에서 1번째로 처리헀을 때, KT proxy가 아니라 SKT proxy를 사용했었는데, https://github.com/AdguardTeam/CoreLibs/issues/1789#issuecomment-1865538317 에서 언급된 설정을 사용해보시곘어요?

![image](https://github.com/List-KR/community/assets/98787049/a257ea71-6570-4972-8e6a-91fa104f13b1)

**Reply by swbaek81** [reply]
해결될때 까지 기다려 보겠습니다. 애드가드 측에 이슈 좀 부탁드립니다. 저는 유선 인터넷 윈도우OS 환경이니까요

**Reply by swbaek81** [reply]
캡쳐 주신 딜레이만 50이 디폴트 값이길래 100으로 변경해서 해봤는데 여전히 ng네요

**Reply by piquark6046** [reply]
@swbaek81 

아래 첨부된 스크린샷처럼 설정해보신 다음에 다시 해당 웹 사이트에 연결해보시겠어요?
https://github.com/AdguardTeam/CoreLibs/issues/1789#issuecomment-1865538317 에 언급된 설정입니다.

<details><summary>Screenshots</summary>

![Screenshot from 2023-12-27 22-39-47](https://github.com/List-KR/community/assets/98787049/8c2fdc08-2aea-4a3c-aa01-857aee4d254b)
![Screenshot from 2023-12-27 22-40-17](https://github.com/List-KR/community/assets/98787049/f7bc75d3-76b4-4b86-8133-3144514747ac)
![Screenshot from 2023-12-27 22-40-36](https://github.com/List-KR/community/assets/98787049/810dccd6-fc2a-4fa6-89ab-dd4f9f20ba7c)

</details>

**Reply by swbaek81** [reply]
캡처대로 해봤는데 ng 네요

**Reply by piquark6046** [reply]
https://github.com/AdguardTeam/CoreLibs/issues/1789#issuecomment-1870322772 남겼습니다. AdGuard Slack에도 남기겠습니다.

**Reply by swbaek81** [reply]
네 근데 사실 인터넷 망이 유선 케이블 망(SK브로드밴드)이라 SKT 통신사랑은 상관이 없는거 아닌가 싶습니다. 일반 유선 케이블 환경에서는 다 안되지 않을까 싶습니다.

**Reply by swbaek81** [reply]
SK텔레콤 으로 적혀잇는거 같길래 노파심에 적었습니다

**Reply by piquark6046** [reply]
> 일반 유선 케이블 환경에서는 다 안되지 않을까 싶습니다.

제가 아까 테스트했을 때는 LG U+ 유선 네트워크였습니다.
